client/src/views/page-components/account/AccountHistoryModal/AccountHistoryModal.ts

```diff
@@ -0,0 +1,388 @@
+import { $, createDOMWithSelector, removeOthersClassList } from '@src/utils/helper';
+import './AccountHistoryModal.scss';
+import handleEvent from '@src/utils/handleEvent';
+import { categoryList, matchCategoryAndImg } from '@src/static/constants';
+import PayMethods from '@src/views/components/PayMethods/PayMethods';
+import { samplePay } from '@src/dummyData';
+
+const slideOutTime: number = 1300;
+const dateInputMaxLength: number = 8;
+const moneyInputMaxLength: number = 9;
+const moneyInputMinLength: number = 0;
+const alertShowTime: number = 2000;
+
+/**
+    추가적으로 고려할 부분 : 
+    1.수입 지출!
+    2.윤년 고려!
+ */
+export default class AccountHistoryModal {
+  state: any;
+  choicedCategoryIndex: number = 0;
+  payMethod: any;
+  modal: any;
+  modalWrapper;
+  dateInput;
+  moneyInput;
+  dateValueValidation: boolean = false;
```

Comment:
boolean 멤버 변수라면 요게 더 나을듯 합니다 😄 

```suggestion
  isDateValueValid: boolean = false;
```